tests/Elasticsearch/Tests/ClientIntegrationTests.php

```diff
@@ -30,8 +54,85 @@ public function testCustomQueryParams()
             'custom' => ['customToken' => 'abc', 'otherToken' => 123],
             'client' => ['ignore' => 400]
         ];
-        $exists = $client->exists($getParams);
+        $exists = self::$client->exists($getParams);
 
         $this->assertFalse($exists);
     }
+
+    /**
+     * @dataProvider aliasDataProvider
+     */
+    public function testExistAlias($alias)
+    {
+        $this->createIndexWithAlias($alias);
+
+        $params = array(
+            'name' => $alias
+        );
+
+        $this->assertTrue(self::$client->indices()->existsAlias($params));
+    }
+
+    /**
+     * @dataProvider aliasDataProvider
+     */
+    public function testFindIndexByAlias($alias)
+    {
+        $this->createIndexWithAlias($alias);
+
+        $params = array(
+            'name' => $alias
+        );
+        $this->assertEquals($alias . '_v1', array_keys(self::$client->indices()->getAlias($params))[0]);
```

Comment:
`$this->assertSame()` should be used insted of `assertEquals` (there are more occurences in the PR)

`assertSame` is like `===`
`assertEquals` is like `==`

Reply:
thanks for the tips, i didn't know that :)